scripts/shared/lib/utils

```diff
@@ -53,37 +53,33 @@ function with_context() {
 }
 
 # Run cluster commands in parallel.
-# 1st argument is the numbers of the clusters to run for, supports "1 2 3" or "{1..3}" for range
+# 1st argument is the cluster names for which to run.
 # 2nd argument is the command to execute, which will have the $cluster variable set.
 # 3rd argument and so forth get passed to the command.
 function run_parallel() {
-    local clusters cmnd
-    clusters=$(eval echo "$1")
-    cmnd=$2
+    local cmnd=$2
     declare -A pids
-    for i in ${clusters}; do
+    for cluster in $(eval echo "$1"); do
         (
            set -o pipefail
-           with_context "cluster${i}" "$cmnd" "${@:3}" | sed "s/^/[cluster${i}] /"
+           with_context "${cluster}" "$cmnd" "${@:3}" | sed "s/^/[${cluster}] /"
```

Comment:
```suggestion
           $cmnd "${@:3}" | sed "s/^/[${cluster}] /"
```

No need to call `with_context` wince you've already set cluster

Reply:
Not sure I follow - where have we already done the equivalent of `with_context`?

Reply:
`for cluster in` already has set `$cluster`

Reply:
OK - I'm not really clear how `with_context` works - the cryptic "${@:3}" syntax is a bit above my rudimentary scripting knowledge but I'll take your word for it 😄